src/main/java/seedu/momentum/model/timer/Time.java

```diff
@@ -31,6 +33,26 @@ public Time(LocalDateTime time) {
         this.time = time;
     }
 
+    /**
+     * Returns a new Time that is after this Time be a specified amount
+     * @param amount Amount to increase by.
+     * @param unit Unit to increase with.
+     * @return
+     */
+    public Time plus(long amount, ChronoUnit unit) {
+        return new Time(time.plus(amount, unit));
+    }
+
+    /**
+     * Returns a new Time that is before this Time by a specified amount.
+     * @param amount Amount to decrease by.
+     * @param unit Unit to decrease with.
+     * @return
+     */
+    public Time minus(long amount, ChronoUnit unit) {
+        return new Time(time.minus(amount, unit));
+    }
```

Comment:
There should be an extra line before `@params` tag and a description for what is returned after the `@return` tag